data/src/main/java/com/stop/data/remote/network/WsBusApiService.kt

```diff
@@ -13,7 +15,16 @@ internal interface WsBusApiService {
         @Query("resultType") resultType: String = "json",
     ): NetworkResult<GetBusStationArsIdResponse>
 
+    @GET(GET_BUS_NOW_LOCATION_URL)
+    suspend fun getBusNowLocation(
+        @Query("ServiceKey") serviceKey: String = BuildConfig.BUS_KEY,
+        @Query("busRouteId") busRouteId: String,
+        @Query("resultType") resultType: String = JSON
+    ): NetworkResult<GetBusNowLocationResponse>
```

Comment:
```suggestion
    @GET(GET_BUS_NOW_LOCATION_URL)
    suspend fun getBusNowLocation(
        @Query("busRouteId") busRouteId: String,
        @Query("resultType") resultType: String = JSON
    ): NetworkResult<GetBusNowLocationResponse>
```

NetworkModule에서 CustomInterceptor가 baseUrl에 따라 serviceKey를 넣어주고 있기 때문에
serviceKey 파라미터는 지우셔도 됩니다🙂